specification/network/resource-manager/Microsoft.Network/stable/2024-07-01/expressRouteCircuit.json

```diff
@@ -1678,6 +1678,45 @@
       },
       "description": "Response for ListAuthorizations API service call retrieves all authorizations that belongs to an ExpressRouteCircuit."
     },
+    "AdvertisedPublicPrefixProperties": {
+      "type": "object",
+      "properties": {
+        "prefix": {
+          "type": "string",
+          "description": "Advertised Public Prefix."
+        },
+        "validationId": {
```

Comment:
>"validationId": { [](http://example.com/codeflow?start=8&length=17)

mark it readOnly if it is generated by the service and not updatable by the client

Reply:
There might be cases where client could pass this field. For example, if cx wants to create a new peering and have the correct format for the validationId and signature, they could pass this in the API. Keeping this as read/write field.